### PR TITLE
Properly adds a Janitor ERT

### PR DESCRIPTION
**Pull request by VileBeggar**
## About The Pull Request

Does what the title basically says, adds a Janitor ERT for admins to use.
There are two types of janitors, normal ones and heavy duty ones:
* Normal janitors just get the stuff that you'd expect janitors to get, a mop, a bucket, some cleaning grenades and yadi-yadi-yada.
* Heavy duty janitors get a fancy wicked chem sprayer which contains space cleaner and regenerates some of it on it's own and can contain more juice, BUT it can't be refueled in any other way, so if the crew gets their hands on it, there won't be a lubeopocalyspe.

## Why It's Good For The Game
As I said before, I've seen admins use these, and it's a shame there isn't a preset for it, hence this PR.

## Changelog
:cl: Vile Beggar
add: Central Command now has access to a janitorial ERT. Be wary of the heavy-duty ones!
/:cl:

### CODE REVIEW

**Comment by actioninja** [comment]
Based and cleanpilled

**Comment by Dorsisdwarf** [comment]
THANK you

**Comment by Bombpants** [comment]
I love you

File: code/datums/ert.dm [hunk 1]
```diff
@@ -53,3 +53,12 @@
...
 	polldesc = "a Nanotrasen paranormal response team"
+
+/datum/ert/janitor
+	roles = list(/datum/antagonist/ert/janitor, /datum/antagonist/ert/janitor/heavy)
+	leader_role = /datum/outfit/ert/janitor/heavy
+	teamsize = 4
+	opendoors = FALSE
+	rename_team = "Janitor"
+	mission = "Clean up EVERYTHING."
+	polldesc = "a Nanotrasen janitorial response team"
```
Comment (ShizCalev) [inline code/datums/ert.dm]: Capitalize response

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/datums/ert.dm (hunk 1 above)
Comment (VileBeggar) [reply code/datums/ert.dm]: for what reason though? it's not capitalized in other places

Comment (ExcessiveUseOfCobblestone) [reply code/datums/ert.dm]:
```suggestion
	polldesc = "a Nanotrasen Janitorial Response Team"
```
https://github.com/tgstation/tgstation/pull/42659/files#diff-d99f27400b888b7587642f5bdfea3770R36
https://github.com/tgstation/tgstation/pull/42659/files#diff-d99f27400b888b7587642f5bdfea3770R45

Comment (VileBeggar) [reply code/datums/ert.dm]: guess i'm blind

File: code/modules/reagents/reagent_containers/spray.dm [hunk 2]
```diff
@@ -309,6 +309,40 @@
...
+	return ..()
+
+/obj/item/reagent_containers/spray/chemsprayer/janitor/process()
+	if(world.time < last_generate + generate_delay)
+		return
+	last_generate = world.time
+	generate_reagents()
+
+/obj/item/reagent_containers/spray/chemsprayer/janitor/proc/generate_reagents()
+	reagents.add_reagent(generate_type, generate_amount)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/reagent_containers/spray.dm]: why make a proc that does 1 thing when you could just add the 1 thing to process?

**Comment by Cyprex** [comment]
I created a janitorial ERT with team size 2, armory doors closed. One of the two spawned in the area but without equipment, they did not have the antag status either(They did not show up in check antagonists)

**Pull request merged**